pages/index.tsx

```diff
@@ -106,14 +114,60 @@ export default function Home() {
     balanceStatuses[balanceStatuses.length - 1]
   )
 
+  const monthStartDate = startOfMonth(currentDate)
+  const monthEndDate = endOfMonth(currentDate)
+  const currentMonthOccurences = generateTransactionConfigsOccurances(
+    transactions,
+    monthStartDate,
+    monthEndDate
+  )
+
+  const yearStartDate = startOfYear(currentDate)
+  const yearEndDate = endOfYear(currentDate)
+  const currentYearOccurences = generateTransactionConfigsOccurances(
+    transactions,
+    yearStartDate,
+    yearEndDate
+  )
+
+  const currentYearBalanceAmount = currentYearOccurences.reduce((res, cur) => {
+    return res + cur.amount
+  }, 0)
+  const currentMonthBalanceAmount = currentMonthOccurences.reduce(
+    (res, cur) => {
+      return res + cur.amount
+    },
+    0
+  )
+
```

Comment:
please move it to `transactionsCalculator.ts`. i think it makes sense to use `getTransactionsSummeryByPeriod` and not `generateTransactionConfigsOccurances`

Reply:
can't really use it since it accepts very different parameters